### DISCUSSION TITLE
SignalR Version

### DISCUSSION THREAD

**Discussion by ogrnurullahcelik** [Benedicht/com.tivadar.best.documentation#4] [Q&A] [answered]
Which version we can use of signalR?

**Comment by Benedicht** [comment, marked as answer]
The plugin supports ASP.NET **Core** SignalR. There's no specific version you have to use, the protocol didn't change too much.
One exception is the [statful reconnect](https://github.com/Benedicht/BestHTTP-Issues/issues/193) support that comes in the next release and requires ASP.Net Core 8.
